src/xmlsec/__init__.py

```diff
@@ -22,6 +22,8 @@
 DS = ElementMaker(namespace=NS['ds'], nsmap=NSDefault)
 
 
+log = logging.getLogger('xmlsec')
```

Comment:
The convention is to use
```python
logger = logging.getLogger(__name__)
```
`__name__` is the module’s name in the Python package namespace.

Either way, I think it's fine.